### PR TITLE
header was commited

### PR DESCRIPTION
**Pull request by RomSmile**
- DEMO LINK: https://RomSmile.github.io/layout_moyo-header/
- TEST REPORT LINK: https://RomSmile.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

**Comment by RomSmile** [comment]
http://joxi.ru/brRxx3ZcLygkL2

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  left: 0;
+  right: 0;
+  content: "";
+  background-color: #00acdc;
+  border-radius: 8px;
+  height: 4px;
+}
+
+.nav_link {
+  margin: 0 10px;
```
Comment (varseniuk) [inline src/style.css:43]: You could use margin-right: 20px, and margin-right: 0 for the :last-chil

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  padding: 23px 0;
```
Comment (varseniuk) [inline src/style.css:44]: You could center the text using display: flex and align-items:center

**Review by varseniuk — APPROVED** [review]
Your page hasn't been deployed. It's a problem. You should have solved it. In the future it's obligatory
Now I left some comments. Pay attention to them

**Pull request closed without merging**